chemprop/cli/predict.py

```diff
@@ -170,9 +170,15 @@ def main(args):
     multicomponent = n_components > 1
 
     if multicomponent:
-        model = MulticomponentMPNN.load_from_checkpoint(args.checkpoint)
+        if args.checkpoint.endswith(".pkl"):
+            model, input_scalers, output_scaler = load_model(args.checkpoint) # TODO: connect input_scalers and output_scaler to the model
+        else:
+            model = MulticomponentMPNN.load_from_checkpoint(args.checkpoint)
     else:
-        model = MPNN.load_from_checkpoint(args.checkpoint)
+        if args.checkpoint.endswith(".pkl"):
+            model, input_scalers, output_scaler = load_model(args.checkpoint) # TODO: connect input_scalers and output_scaler to the model
+        else:
+            model = MPNN.load_from_checkpoint(args.checkpoint)
```

Comment:
we should have separate concepts of a `checkpoint` and a `saved_model`. The point of making a model state a subset of a checkpoint allows for users to simply supply a checkpoint file as the value of `--model-path`

Reply:
Thanks, that makes sense! I moved the `--checkpoint` and other related arguments from `common.py` to `train.py`, added `--model-path` to `predict.py`, and modified relevant codes to use `--model-path` in `predict.py`.